README.md

```diff
@@ -329,6 +335,7 @@ Alchemy specs are written **in RSpec 3**. Please **do not use deprecated RSpec 2
 * API Documentation: <http://rubydoc.info/github/AlchemyCMS/alchemy_cms>
 * Issue-Tracker: <https://github.com/AlchemyCMS/alchemy_cms/issues>
 * Sourcecode: <https://github.com/AlchemyCMS/alchemy_cms>
+* Slack: <https://alchemy-cms.slack.com>
```

Comment:
Would you please replace the IRC with Slack?

Reply:
I do not understand what would you like to change...
Should I delete the IRC line in total?
Should I keep the IRC and put the Slack address in place of the IRC address?
Should I move the Slack line in the "place" of IRC line and delete the IRC line?
Please advise
Thank you

Reply:
Like I said. Replace the IRC line with a link to Slack.

Reply:
Done...
I would suggest changing also the label...

Reply:
Done...
Changed also the label...